### DISCUSSION TITLE
Plugin not found: grafana-cli plugins install

### DISCUSSION THREAD

**Discussion by webstercharly** [grafana/grafana-infinity-datasource#33] [Q&A] [answered]
Is this a known problem? Didn't want to raise as an Issue if it was.

Step 17/35 : RUN grafana-cli plugins install yesoreyeram-infinity-datasource
 ---> Running in 3390889382f0
Error: ✗ Failed to find requested plugin, check if the plugin_id is correct: 404 not found error
The command '/bin/sh -c grafana-cli plugins install yesoreyeram-infinity-datasource' returned a non-zero code: 1

**Comment by yesoreyeram** [comment]
Plugin is not yet published to grafana plugin store. Soon it will be. Checkout install instructions om how to install plugin. BTW plugin is signed and ready to used.

**Comment by yesoreyeram** [comment, marked as answer]
Infinity plugin is now available in [grafana.com plugins page](https://grafana.com/grafana/plugins/yesoreyeram-infinity-datasource/). You can install from there into your instance of grafana.
